sk-csharp-chatgpt-plugin/azure-function/azure-function.csproj

```diff
@@ -0,0 +1,58 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
Would be better to have a more specific name `sk-chatgpt-azure-function`

Reply:
fixed :)